colossalai/inference/core/request_handler.py

```diff
@@ -13,8 +15,9 @@ class RequestHandler:
     def __init__(self, cache_config) -> None:
         self.cache_config = cache_config
         self._init_cache()
-        self.waiting_list: List["Reqseq"] = []
-        self.running_list: List["Reqseq"] = []
+        self.waiting_list: List["Sequence"] = []
+        self.running_list: List["Sequence"] = []
+        self.batch = BatchHandler.init_batch([])
```

Comment:
This method call is pretty weird to me, `init_batch` receives an empty list as input. If the arg is an empty list, the default arg value should be None.